node-fs-readfile/cat.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+
+console.log(process.argv);
+
+for (let i = 2; i < process.argv.length; i++) {
+  try {
+    const filename = process.argv[i];
+    fs.readFile(filename, 'utf8', (err, data) => {
+      if (err) throw err;
+      console.log(data);
+    });
+  } catch (err) {}
```

Comment:
Why the `try/catch`? If there's an error reading the file, unfortunately, the error will not be thrown synchronously. You will need a mind shift for this one as you cannot guarantee that the reads will complete in the order you start them.